packages/playground/build.ts

```diff
@@ -1,20 +1,249 @@
+import lockfile from "proper-lockfile";
```

Comment:
There's no default export `lockfile` in `proper-lockfile`.

This should probably be imported as
```
import { lockSync, unlockSync } from "proper-lockfile";
```
with the necessary adjustments where such methods are used.